src/screens/ProfileScreen/elements/KarmaLock.tsx

```diff
@@ -28,7 +28,7 @@ export const KarmaLock = (props: {onPressCreatePost: () => void}) => {
         <Text
           style={{
             fontSize: 12,
-            color: COLORS.signed_primary,
+            color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from `COLORS.signed_primary` to `COLORS.white2`. Ensure that this color change aligns with the overall design and accessibility guidelines, as color contrast can impact readability for users.
```diff
- color: COLORS.signed_primary,
+ color: COLORS.white2,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->